### PR TITLE
Option to find single range when hitfinding

### PR DESCRIPTION
**Pull request by JelleAalbers**
This is a small extra feature for the hitfinder useful for gain calibration and/or hitfinder performance testing.

If you specify:
```
[HitFinder]
always_find_single_hit=(10, 20)
```
then the hitfinder will "find" a single hit between 10 and 20 (inclusive) samples in each pulse.

### CODE REVIEW

**Comment by ErikHogenbirk** [comment]
For the record, I requested this (thank you Jelle! :) ) so if you disagree you can direct your complaints to me.  I think this is a useful and nice and small feature.

File: pax/plugins/signal_processing/HitFinder.py [hunk 1]
```diff
@@ -291,10 +299,16 @@ def build_hits(w, hit_bounds,
                 saturation_count += 1
             area += x
             center += x * i
-        center /= area
-        for i, x in enumerate(w[left:right + 1]):
-            deviation += x * abs(i - center)
-        deviation /= area
+
```
Comment (tunnell) [inline pax/plugins/signal_processing/HitFinder.py]: __[flake8]__
* [W293] blank line contains whitespace

File: pax/plugins/signal_processing/HitFinder.py [hunk 2]
```diff
@@ -291,10 +299,16 @@ def build_hits(w, hit_bounds,
                 saturation_count += 1
             area += x
             center += x * i
-        center /= area
-        for i, x in enumerate(w[left:right + 1]):
-            deviation += x * abs(i - center)
-        deviation /= area
+            
+        # During gain calibration, or if the low threshold is set to negative values,
```
Comment (tunnell) [inline pax/plugins/signal_processing/HitFinder.py]: __[flake8]__
* [W291] trailing whitespace

**Comment by JelleAalbers** [comment]
@ErikHogenbirk or @arjenwildeboer could either of you approve this if you think it is ok?

**Comment by ErikHogenbirk** [comment]
We have been using this successfully the past few weeks. I'd say merge (though I am not autorized to do so on GitHub).

**Pull request merged**